modules/olmv1-deleting-an-operator.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc
+
+:_content-type: PROCEDURE
+
+[id="olmv1-deleting-an-operator_{context}"]
+= Deleting an Operator
+
+You can delete an Operator and its custom resource definitions (CRDs) by deleting the Operator's custom resource (CR).
+
+.Prerequisites
+
+* You have a catalog installed.
+* You have an Operator installed.
+
+.Procedure
+
+* Run the following command to delete an Operator and its CRDs:
++
+[source,terminal]
+----
+$ oc delete operator.operators.operatorframework.io quay-example
+----
++
+.Example output
+[source,text]
+----
+operator.operators.operatorframework.io "quay-example" deleted
+----
+
+.Verification
+
+* Run the following commands to verify that your Operator was deleted:
++
+[source,terminal]
+----
+$ oc get operator.operators.operatorframework.io
+----
++
+[source,terminal]
+----
+$ oc get pod -n quay-operator-system
+----
++
+.Example output
```

Comment:
@michaelryanpeter there are two commands which have two outputs, but your code only one output.
here is the output I tried.
```console
[root@preserve-olm-env2 olmv1review]# oc get operator.operators.operatorframework.io
No resources found
[root@preserve-olm-env2 olmv1review]# oc get pod -n quay-operator-system
No resources found in quay-operator-system namespace.
[root@preserve-olm-env2 olmv1review]# 
```

actually quay-operator-system ns is also deleted, there is no need `oc get pod -n quay-operator-system`

so, 
1, remove the command `oc get pod -n quay-operator-system`
2, if you want to keep it,  better to update the output per command.
3, change `oc get pod -n quay-operator-system` to `oc get ns quay-operator-system`
```console
[root@preserve-olm-env2 olmv1review]# oc get ns quay-operator-system
Error from server (NotFound): namespaces "quay-operator-system" not found
```

I prefer to 3.

Reply:
@kuiwang02: PTAL

Reply:
it is ok. thanks